src/redux/saga/employer.ts

```diff
@@ -1,19 +1,55 @@
-import { put, takeLatest } from 'redux-saga/effects'
+import { put, takeLatest, select } from 'redux-saga/effects'
 import axios from 'axios'
 
-import { registerEmployerSuccess } from '../../redux/actions/employer'
+import { AppState } from '../types'
 
-function* registerEmployerSaga(credentials: Credential) {
+import {
+  registerEmployerSuccess,
+  registerEmployerFail,
+  loginEmployerSuccess,
+  loginEmployerFail,
+} from '../../redux/actions/employer'
+
+const credential = (state: AppState) => state.employer.credential
+function* registerEmployerSaga() {
   try {
-    const req = yield axios.post('/employer', { credentials })
+    const credentialData = yield select(credential)
+    const req = yield axios.post('/employer', {
+      info: {},
+      credential: credentialData,
+    })
     yield put(registerEmployerSuccess(req.data))
   } catch (error) {
-    console.log(error)
+    yield put(registerEmployerFail())
+  }
+}
+
+function* loginEmployerSaga() {
+  try {
+    const credentialData = yield select(credential)
+    const res = yield axios.post('/employer/login/local', {
+      email: credentialData.email,
+      password: credentialData.password,
+    })
+
+    yield put(loginEmployerSuccess(res))
+    localStorage.setItem('employerInfo', JSON.stringify(res.data))
```

Comment:
I already made some `localStorage` method in the `localStorage` folder, can you check that out and use them instead

Reply:
Ok.